src/components/Expander/Expander.jsx

```diff
@@ -0,0 +1,152 @@
+import _ from 'lodash';
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition }  from '../../util/component-definition';
+import ChevronIcon from '../Icon/ChevronIcon/ChevronIcon';
+import * as reducers from './Expander.reducers';
+
+const boundClassNames = lucidClassNames.bind('&-Expander');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"], "madeFrom": ["ChevronIcon"]}
+ *
+ * This is a container that provides a toggle that controls when the content is
+ * shown.
+ */
+const Expander = React.createClass(createLucidComponentDefinition({
+	displayName: 'Expander',
+
+	childProps: {
+		Label: {
+			/**
+			 * Used to identify the purpose of this switch to the user -- can be
+			 * any renderable content.
+			 */
+			children: node
+		}
+	},
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Expandable content.
+		 */
+		children: node,
+
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+
+		/**
+		 * Indicates that the component is in the "expanded" state when true
+		 * and in the "unexpanded" state when false.
+		 */
+		isExpanded: bool,
+
+		/**
+		 * Called when the user clicks on the component's header.
+		 *
+		 * Signature: `(isExpanded, { event, props }) => {}`
+		 */
+		onExpand: func,
+
+		/**
+		 * Passed through to the root element.
+		 */
+		style: object,
+
+		/**
+		 * Child element whose children represents content to be shown next to
+		 * the expander icon.
+		 */
+		Label: any
```

Comment:
we do still need this for the time being, i need make the docs parsing better to support lucid child components